### PR TITLE
Added new endpoint api/v1/status

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

**Review by ondratu — APPROVED** [review]

File: spec/openapi.yaml [hunk 1]
```diff
@@ -963,6 +980,91 @@ components:
           type: string
           example: "/dev/tty"
 
+    StatusJob:
+      type: object
+      description: Telemetry info about current job, all values are optional
+      properties:
+        id:
```
Comment (Lowentwickler) [inline spec/openapi.yaml:987]: [question]
What is that ID? Is it possible to have there path to the file that is being printing?

File: spec/openapi.yaml [hunk 2]
```diff
@@ -963,6 +980,91 @@ components:
...
+          example: 526
+
+    StatusPrinter:
+      type: object
+      description: Telemetry info about printer, all values except state are optional
+      properties:
+        state:
+          type: string
+          enum: [ IDLE, BUSY, PRINTING, PAUSED, FINISHED, STOPPED, ERROR, ATTTENTION, READY ]
+        temp_nozzle:
```
Comment (Lowentwickler) [inline spec/openapi.yaml:1007]: Will there target temperatures? Nozzle Size?

File: spec/openapi.yaml [hunk 3]
```diff
@@ -963,6 +980,91 @@ components:
...
+        speed:
+          type: integer
+          example: 100
+        fan_hotend:
+          type: integer
+          example: 420
+        fan_print:
+          type: integer
+          example: 420
+        connect_status:
```
Comment (Lowentwickler) [inline spec/openapi.yaml:1036]: What about serial port status?

**Review by Lowentwickler — COMMENTED** [review]
I would love to see here also something like list of available storages. Could be without free/used space, but at least to know if USB(one or many) is inserted

File: spec/openapi.yaml (hunk 2 above)
Comment (Majkluss) [reply spec/openapi.yaml:1007]: Nozzle size is part of api/v1/info, I can add target temperatures

File: spec/openapi.yaml (hunk 1 above)
Comment (Majkluss) [reply spec/openapi.yaml:989]: Sure, I'll add path

File: spec/openapi.yaml (hunk 3 above)
Comment (Majkluss) [reply spec/openapi.yaml:1047]: As we spoke before, I'll add section printer_status, which includes the serial port status.

**Review by HelclM — APPROVED** [review]

**Review by ondratu — APPROVED** [review]

**Pull request merged**